data/json/mapgen/animalpound.json

```diff
@@ -0,0 +1,161 @@
+[
+  {
+    "name": "GROUP_ANIMALPOUND_CATS",
+    "type": "monstergroup",
+    "default": "mon_null",
+    "monsters": [
+      { "monster": "mon_cat", "freq": 300, "cost_multiplier": 0 }
+    ]
+  },
+  {
+    "name": "GROUP_ANIMALPOUND_DOGS",
+    "type": "monstergroup",
+    "default": "mon_dog_zombie_rot",
+    "monsters": [
+      { "monster": "mon_zombie_dog", "freq": 250, "cost_multiplier": 0 },
+      { "monster": "mon_dog_zombie_rot", "freq": 175, "cost_multiplier": 0 },
+      { "monster": "mon_dog", "freq": 75, "cost_multiplier": 0 },
+      { "monster": "mon_dog_bull", "freq": 75, "cost_multiplier": 0 },
+      { "monster": "mon_dog_pitbullmix", "freq": 100, "cost_multiplier": 0 },
+      { "monster": "mon_dog_chihuahua", "freq": 75, "cost_multiplier": 0 },
+      { "monster": "mon_dog_gshepherd", "freq": 35, "cost_multiplier": 0 }
+    ]
+  },
+  {
+    "id": "animalpound",
+    "type": "overmap_terrain",
```

Comment:
Not saying there aren't any, but I haven't seen any mapgen files with the overmap_terrain defined within them. I doubt it would cause an error but for organizational purposes I may be better to define them in overmap_terrain.json.

Reply:
There are several such locations:

```
  C:\Projects\Cataclysm-DDA\data\json\mapgen\animalpound.json (1 hit)
	Line 24:     "type": "overmap_terrain",
  C:\Projects\Cataclysm-DDA\data\json\mapgen\animalshelter.json (1 hit)
	Line 117:     "type": "overmap_terrain",
  C:\Projects\Cataclysm-DDA\data\json\mapgen\antique_store.json (1 hit)
	Line 4:     "type": "overmap_terrain",
  C:\Projects\Cataclysm-DDA\data\json\mapgen\arcade.json (1 hit)
	Line 10:     "type": "overmap_terrain",
  C:\Projects\Cataclysm-DDA\data\json\mapgen\bowling_alley.json (1 hit)
	Line 139:     "type": "overmap_terrain",
  C:\Projects\Cataclysm-DDA\data\json\mapgen\boxing.json (1 hit)
	Line 57:     "type": "overmap_terrain",
  C:\Projects\Cataclysm-DDA\data\json\mapgen\dojo.json (1 hit)
	Line 71:     "type": "overmap_terrain",
  C:\Projects\Cataclysm-DDA\data\json\mapgen\fire_station.json (1 hit)
	Line 25:     "type": "overmap_terrain",
  C:\Projects\Cataclysm-DDA\data\json\mapgen\gardening_store.json (1 hit)
	Line 4:     "type": "overmap_terrain",
  C:\Projects\Cataclysm-DDA\data\json\mapgen\gym.json (1 hit)
	Line 36:     "type": "overmap_terrain",
  C:\Projects\Cataclysm-DDA\data\json\mapgen\homeimprovement.json (1 hit)
	Line 23:     "type": "overmap_terrain",
  C:\Projects\Cataclysm-DDA\data\json\mapgen\jewel_store.json (1 hit)
	Line 4:     "type": "overmap_terrain",
  C:\Projects\Cataclysm-DDA\data\json\mapgen\laundromat.json (1 hit)
	Line 19:     "type": "overmap_terrain",
  C:\Projects\Cataclysm-DDA\data\json\mapgen\mapgen-test.json (1 hit)
	Line 19:     "type": "overmap_terrain",
  C:\Projects\Cataclysm-DDA\data\json\mapgen\mortuary.json (1 hit)
	Line 39:     "type": "overmap_terrain",
  C:\Projects\Cataclysm-DDA\data\json\mapgen\museum.json (1 hit)
	Line 4:     "type": "overmap_terrain",
  C:\Projects\Cataclysm-DDA\data\json\mapgen\veterinarian.json (1 hit)
	Line 100:     "type": "overmap_terrain",
  C:\Projects\Cataclysm-DDA\data\json\overmap\multitile_buildings_terrain.json (2 hits)
	Line 3:     "type" : "overmap_terrain",
	Line 13:     "type" : "overmap_terrain",
```

I guess we should move them to `overmap_terrain.json`, but in separate PR.